### PR TITLE
fix: Emit notification on mentions

### PR DESCRIPTION
**Pull request by juliushaertl**
It seems we totally missed this part when implementing the initial mention support back in the days.

This will emit a notification to mentioned users that can be clicked, just like it works in text.

### CODE REVIEW

File: lib/Controller/MentionController.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+		$notification = $this->manager->createNotification();
+		$notification->setUser($mention)
+			->setApp(Application::APPNAME)
+			->setSubject(Notifier::TYPE_MENTIONED, [
+				Notifier::SUBJECT_MENTIONED_SOURCE_USER => $this->userId,
+				Notifier::SUBJECT_MENTIONED_TARGET_USER => $mention,
+			])
+			->setObject('file', (string)$fileId);
+		;
```
Comment (elzody) [inline lib/Controller/MentionController.php:55]: random semicolon maybe?

Comment (juliushaertl) [reply lib/Controller/MentionController.php]: Right, fixed 👍

**Review by elzody — APPROVED** [review]

**Pull request merged**